react/useVideo.tsx

```diff
@@ -89,7 +90,8 @@ const useVideo = (
     setState({
       ...state,
       isMuted: currentVideoRef?.muted === true,
-      isPlaying: currentVideoRef?.autoplay || currentVideoRef?.paused === false,
+      isPlaying:
+        Boolean(currentVideoRef?.autoplay) || currentVideoRef?.paused === false,
```

Comment:
```suggestion
        !!currentVideoRef?.autoplay || currentVideoRef?.paused === false,
```

Reply:
I would maintain `Boolean`, I personally think it's more readable